include/vfio-user.h

```diff
@@ -133,6 +133,20 @@ struct vfio_user_irq_info {
     uint32_t    subindex;
 } __attribute__((packed));
 
+/* based on struct vfio_bitmap */
```

Comment:
do we still need vfio_bitmap definitions, line 188 onwards?

Reply:
Good point, we only need `vfio_iommu_type1_dirty_bitmap`.